client/src/parent/chores/AddChoreDialog.tsx

```diff
@@ -86,15 +86,17 @@ export function AddChoreDialog(props: Prop) {
                 }
               };
 
-              const errors: Partial<ChoreValidation> = {};
-              if (!ChoreValidation.choreName) {
-                errors.choreName = "Bitte definieren";
-              } else if (!ChoreValidation.choreValue) {
-                errors.choreValue = "Bitte definieren";
-              } else if (ChoreValidation.choreValue < 1) {
-                errors.choreValue = "Ein wenig unfair, nicht?";
-              } else if (!validDate(ChoreValidation.date, new Date())) {
-                errors.date = validDateMessage;
+              const errors: Partial<AddChoreValidation> = {};
```

Comment:
Vielleicht könnten wir für die Validierung die [Yup Library](https://github.com/jquense/yup) verwenden?
Aber das kann man auch in einem späteren Schritt noch tun, falls man das will.